chatmeleon-server/tsconfig.json

```diff
@@ -1,16 +1,16 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "module": "esnext",
+    "module": "NodeNext",
```

Comment:
@WataNekko already mentioned this and some of problem with import formats, this needs to be addressed and agreed upon